app/src/main/java/com/beyondidentity/authenticator/sdk/android/SignInActivity.kt

```diff
@@ -13,8 +13,8 @@ class SignInActivity : AppCompatActivity() {
         val logInButton = findViewById<AuthView>(R.id.auth_view)
 
         logInButton.initAuthView(
-            loginUrl = "$ACME_CLOUD_URL/start",
-            redirectUrl = "${getString(R.string.app_scheme)}://${getString(R.string.app_host)}",
+            authorizationEndpoint = "$ACME_CLOUD_URL/start",
```

Comment:
This would still be something along the lines of `loginUrl`. Only because authorization endpoint refers to the endpoint you would hit if you were only getting the authorization code.

Reply:
That makes sense.